<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: bold 18pt &quot;Calibre&quot;; color:rgb(231, 231, 231)</string>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="lbl_login_email">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>50</y>
            <width>61</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: bold 16px; color:rgb(204, 0, 0)</string>
          </property>
          <property name="text">
           <string>Email</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="txt_login_email">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>50</y>
            <width>181</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(235, 235, 235);</string>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="lbl_login_pass">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>80</y>
            <width>101</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: bold 16px; color:rgb(204, 0, 0)</string>
          </property>
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="txt_login_pass">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>80</y>
            <width>181</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(235, 235, 235);</string>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="btn_login_signin">
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>22</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 14px;
background-color: rgb(235, 235, 235);</string>
               </property>
               <property name="text">
                <string>Sign In</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>110</x>
               <y>10</y>
               <width>114</width>
               <height>16</height>
              </rect>
             </property>
             <property name="maximumSize">
              <size>
               <width>130</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color:rgb(255, 255, 255)</string>
             </property>
             <property name="text">
              <string>Don't Have an account?</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_login_register">
             <property name="geometry">
              <rect>
               <x>230</x>
               <y>10</y>
               <width>40</width>
               <height>16</height>
              </rect>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>20</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    color: white;
    text-decoration: underline;
    border: none;
    background: transparent;
}

QPushButton:hover {
    color: lightblue;
}</string>
             </property>
             <property name="text">
              <string>Register</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
